.github/workflows/classicpress-release.yml

```diff
@@ -0,0 +1,41 @@
+name: Check for ClassicPress release
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
If this runs at 1:00am UTC instead, then it will run < 1 hour after each nightly build finishes, instead of > 23 hours after.

Reply:
PR updated.